crates/api_common/src/utils.rs

```diff
@@ -1162,6 +1162,19 @@ fn build_proxied_image_url(
   ))
 }
 
+pub fn check_code_verifier(code_verifier: &str) -> LemmyResult<&str> {
```

Comment:
There's no need to return the code verifier when you just passed it in. Just return `LemmyResult<()>`